logger.go

```diff
@@ -98,14 +97,16 @@ const (
 	sFatal
 )
 
-type logger struct {
+// A Logger represents an active logging object, multiple loggers can be used
```

Comment:
Somewhat of a nit: I'd make two sentences instead of the run-on/comma-splice without a conjunction.